tools/generator-go-sdk/internal/generator/templater_methods.go

```diff
@@ -500,6 +527,11 @@ func (c methodsPandoraTemplater) requestOptions() (*string, error) {
 		options = "OptionsObject: options,"
 	}
 
+	customPagerOption := ""
+	if c.operation.FieldContainingPaginationDetails != nil {
+		customPagerOption = fmt.Sprintf("Pager: %sCustomPager,", c.operationName)
```

Comment:
Presumably we need to pass in an instance of the `CustomPager` in order for this to work?

```suggestion
		customPagerOption = fmt.Sprintf("Pager: %sCustomPager{},", c.operationName)
```